packages/gatsby-plugin-preact/package.json

```diff
@@ -25,7 +27,8 @@
   "main": "index.js",
   "peerDependencies": {
     "gatsby": "^2.0.0",
```

Comment:
We added refresh webpack plugin recently - so this likely would need to be bumped - and so maybe this plugin need major bump given that it requires newer `gatsby` core?

Reply:
oh nvm, I see you only remove it